google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -530,6 +537,14 @@ private static String detectContentType(StorageObject object, Map<Option, ?> opt
     return firstNonNull(contentType, "application/octet-stream");
   }
 
+  private static Function<String, Bucket> createUnreachableBucket() {
+    return new Function<String, Bucket>() {
+      public Bucket apply(String bucketName) {
+        return new Bucket().setName(bucketName).set("isUnreachable", "true");
+      }
+    };
```

Comment:
nit: this can be much more terse
```suggestion
    return bucketName -> new Bucket().setName(bucketName).set("isUnreachable", "true");
```